dist/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Todo List</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="">
+  </head>
+  <body>
+    <h3>  TODO LIST </h3>
+    <nav></nav>
+    
+    <!-- <div>&nbsp</div> -->
+    <button  id="create_new_project">Create new project</button>
+    
+    <div id="myDIV" style="display: none">
+      <label for="project_title">Project Title:</label><br>
+      <input type="text" id="project_title" name="project_title"><br>
+      <label for="lname">Project Description:</label><br>
+      <input type="text" id="project_description" name="project_description"><br>
+      <button id="save_project">Save Project</button>
+    </div>
+    
+    <!-- <div>&nbsp</div> -->
+    <div id="content"></div>
+    <!-- <div id="myTask" style="display: none">
+      <label for="title">Task Title:</label><br>
+      <input type="text" id="title" name="title"><br>
+      <label for="due_date">Task Due Date:</label><br>
+      <input type="text" id="due_date" name="due_date" ><br>
+      <label for="priority">Priority:</label><br>
+      <input type="text" id="priority" name="priority" ><br>
+      <label for="description">Desciption:</label><br>
+      <input type="text" id="description" name="description"><br><br>
+      <button id="add_task_to_project">Add Task</button>
+      <button id="close_task_pane">Close </button>
+    </div> -->
```

Comment:
## Delete commented out code

- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))